### PR TITLE
Fixed bug with defining params for act method of the notebook plugin.

### PR DESCRIPTION
**Pull request by User1000**
Fixed bug with defining params for act method of the notebook plugin.

### CODE REVIEW

**Review by skndrg — COMMENTED** [review]
Left some comments.

File: tests/test_flow.py [hunk 1]
```diff
@@ -143,10 +160,27 @@ def test_insall_two_notebook_plugins(self):
...
-                .get_args_count("a")
+                .args({"a": "123", "b": "321"})
+                .get_args_count()
         )
 
         # Assert
         self.assertEqual(expected1, actual1)
         self.assertEqual(expected2, actual2)
+
+    def test_insall_notebook_plugins_with_use_of_flow_param(self):
```
[the remark this answers is not in the document]
Comment (User1000) [reply tests/test_flow.py:171]: In theNotebookPluginBase.act method we pass LazyNotebook and Flow. The problem was that flow wasn't Flow but was also LazyNotebook. To check that flow is Flow we need to access its properties. We have similar test but for the FlowPluginBase.

[the remark this answers is not in the document]
Comment (skndrg) [reply tests/test_flow.py:171]: @User1000 Alright. Then, I'd suggest adding an `assert` inside the plugin's method checking for a type of passed arguments.

[the remark this answers is not in the document]
Comment (skndrg) [reply tests/test_flow.py:171]: Also, for the test covering  `FlowPluginBase`.

**Review by skndrg — APPROVED** [review]

**Pull request merged**